### DISCUSSION TITLE
Anchor ID links with next/link?

### DISCUSSION THREAD

**Discussion by michaeljblum** [vercel/next.js#27424] [Help]
Very basic problem: I'd like to be able to use next/link to link to IDs on pages. 

I.e.
```
<Link href="/#about" scroll={false}>
    <a className="header-text">About</a>
</Link>
```
Linking to `<section className="about" id="about">` on click.

Instead, on clicking the Link component with destination "/#about", I am sent to the top of the "/" page, not further down where "/#about" lives. 

I understand I can do something like `onClick={() => router.push("/#about")}` for the link (excuse the inline), which works as expected, but... should I? Do I miss out on SEO/perf optimizations from next/link by doing so?

**Comment by leerob** [comment]
This should work, I do it all the time on my blog. 

https://leerob.io/blog/serverless-redis-nextjs#connecting-to-redis

**Reply by leerob** [reply]
Source: https://github.com/leerob/leerob.io

**Reply by leerob** [reply]
Wait, why do you have `scroll={false}`?

**Reply by michaeljblum** [reply]
Thanks for the reply, Lee. `scroll={false}` is just something I was trying, wondering if it would have an effect. Tried with and without, same result. I wonder why I'm not getting the expected result. Again - the `router.push("/#about")` is working as expected. I'll try navigating to an anchor on a non-index page with an `href` from a `next/link` component. Maybe it just doesn't like those?
